### DISCUSSION TITLE
New Destination: Opensearch

### DISCUSSION THREAD

**Discussion by sivankumar86** [airbytehq/airbyte#36096] [New Connector Request]
## Tell us about the problem you're trying to solve
  Elasticsearch become Opensearch for opensource community hence, Elastic search client library no longer works for opensearch 

https://opensearch.org/docs/latest/clients/java-rest-high-level/

## Describe the solution you’d like
  Want to sync data from s3 /cdc to opensearch in AWS.

## Describe the alternative you’ve considered or used

   Create a python or java program and run it from airflow however, it is difficult for reuse for other project 

## Additional context
   I have tried with elastic search connector however, I got NPE . 

## Are you willing to submit a PR?
   Sure, I  am happy to contribute if someone can review/accept it.

**Comment by misteryeo** [comment]
@sivankumar86 Please feel free to submit a PR on this and the team would be happy to review it!

**Comment by jay-choe** [comment]
@misteryeo can I develop?

**Comment by sivankumar86** [comment]
@jay-choe Please go head

[a comment by an automated account is not shown]
